_projects/SP19-EventHorizon.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: project
+title: Event Horizon
+desc: |
+ A first-person shooter game that's set in space. Authors: yxzhang2, painap2, sujalfs2, blyu3, weihang2
+github: We have a Gitlab link. However, in order not to share in public space, please contact us to allow you to access the Unity project source code.
```

Comment:
This is parsed as a link and will break -- please put special instructions somewhere else. I'll suggest a change, it is up to you how you want to communicate it.

Reply:
```suggestion
```